src/utils/client_server_name.js

```diff
@@ -70,6 +70,7 @@ const clientToServerNames = {
   MICROSOFT_CLARITY: 'Microsoft Clarity',
   SENDINBLUE: 'Sendinblue',
   OLARK: 'Olark',
+  Lemnisk: "Lemnisk",
```

Comment:
This is the mapping from the config name to the display name.
As you have declared `LEMNISK` as the config name in the constants.js file then the mapping should be 
`LEMNISK: "Lemnisk",`

Reply:
Done